emit_main/workflow/l2b_tasks.py

```diff
@@ -317,7 +324,7 @@ def work(self):
         # Copy files to tmp dir and rename
         wm.copy(acq.abun_nc_path, daac_abun_nc_path)
         wm.copy(acq.abununcert_nc_path, daac_abununcert_nc_path)
-        wm.copy(acq.rdn_png_path, daac_browse_path)
```

Comment:
There are a couple more rdn_png_path references in this task that need to be replaced with abun_png_path.  You caught them all on the L2A deliver task, but missed some here.

Reply:
Thanks!  I think these should now be resolved in the commit below.